### ISSUE TITLE
The root option of the automount settings (or maybe just all of wsl.conf) does not work.

### ISSUE THREAD

**Issue by danielmrussell** [MicrosoftDocs/WSL#1717]
### Documentation Issue

When running `wsl --mount` command (without `--bare`) automounts from the Linux side to `/mnt/wsl`. Supposedly setting `root` in /etc/wsl.conf should allow you to change this, but it doesn't work.

The drvfs drives are mounted under /mnt, but how do you set your own mount points?

The wsl.exe command really seriously needs to have a `--mountpoint` option. It would also be great if it had the ability to make virtual hard drives (vhdx files), as that whole process is error prone with other tools, and users should not have to install Hyper-V just to get access to tools with less friction. It would also be nice to be able to format vhdx files from wsl.exe and format them with ext4 in particular.

In other words, wsl.exe should have all the built-in commands needed to make creating, mounting, resizing, and even formatting virtual drives a breeze.

### Link to documentation page

When running wsl.exe --mount, it incorrectly states:

`Note: The location will be different if you have modified the automount.root setting in /etc/wsl.conf.`

### Suggested Improvements

Fix the documentation for now, and more importantly, fix wsl.exe to make all of this stuff easy and reliable.

**Comment by craigloewen-msft** [comment]
@danielmrussell  the /etc/wsl.conf root options only affect auto mounted Windows directories. You can specify mount options etc. with the `wsl --mount` command itself. 

More info here: https://learn.microsoft.com/en-us/windows/wsl/wsl2-mount-disk#specifying-mount-options
